src/jest/config.json

```diff
@@ -15,6 +15,8 @@
   "moduleNameMapper": {
     "^ui_framework/components": "<rootDir>/ui_framework/components",
     "^ui_framework/services": "<rootDir>/ui_framework/services",
+    "^ui/(.*)": "<rootDir>/src/ui/public/$1",
+    "^plugins/kibana/(.*)": "<rootDir>/src/core_plugins/kibana/public/$1",
```

Comment:
I think all of the `plugins/kibana/...`s can be made into local imports, and this be removed(?)